cookbooks/bcpc-hadoop/recipes/graphite_queries.rb

```diff
@@ -196,7 +196,7 @@ def format_disk_size_hash(disk_size_hash)
 
 # monitor all disks on head nodes
 head_nodes_objs.each do |head_node_obj|
-  host = head_node_obj.hostname
+  host = head_node_obj[:hostname]
```

Comment:
string instead of symbol please

Reply:
bot cluster_def and cluster_data stuff cluster.txt fields as symbols and not strings